src/interfaces/swig/shogun.i

```diff
@@ -233,6 +233,10 @@ namespace shogun
 %template(_get_real) CSGObject::get<float64_t, void>;
 %template(_get_int) CSGObject::get<int32_t, void>;
 %template(_get_real_matrix) CSGObject::get<SGMatrix<float64_t>, void>;
+%template(_get_bool_string_list) CSGObject::get<SGStringList<bool>, void>;
```

Comment:
ah yes, here we have the offender. You are getting the CI errors since these methods are only defined within  `#ifdef SWIGPYTHON`. @gf712 can give instructions where else to add this.

BTW once #4470 is merged this will lead to a merge conflict in here. Depends on what is done earlier, but it should be easy to resolve it

Reply:
So now you need to add the method name as a string here and not worry about it.
https://github.com/shogun-toolbox/shogun/blob/13961a1d353414f301370425e124c18a4ce4c3b2/src/interfaces/python/swig_typemaps.i#L1348-L1354
And when you recompile it should be accessible via .get in the Python interface.

Reply:
But you still need the template, which should be `%template(get_bool_string_list)`